### PR TITLE
Fix #61 Machine readable generating keypair's output

### LINKED ISSUE #61: Machine readable generating keypair's output
In stellar, generating keypair is anoying to launch multiple nodes especially in containers.

Currently generating keypair's output is:

```sh
$ sebak key generate
       Secret Seed: SCN4NSV5SVHIZWUDJFT4Z5FFVHO3TFRTOIBQLHMNPAZJ37K5A2YFSCBM
    Public Address: GALQG5SCKCPXUG4ODPMFZJGZ6XBVJTLAJFR7OJKJOJVARA7M4H5SGSOG
```

It is hard to parse in a command line.

If it prints a format which is machine readable like:

```sh
$ sebak key generate --json
{ 
    "secret-seed": "SCN4NSV5SVHIZWUDJFT4Z5FFVHO3TFRTOIBQLHMNPAZJ37K5A2YFSCBM",
    "public-address": "GALQG5SCKCPXUG4ODPMFZJGZ6XBVJTLAJFR7OJKJOJVARA7M4H5SGSOG"
}
```

It's much better to launch nodes in Docker, Docker compose or Kubernetes.

### PR DESCRIPTION
**Pull request by zzim2x**
* fix #61 and I'll use it for #3 ~ #10, if possible.
 * #69 was broken and made agian.

Basic usage

```
$ sebak key generate --format json [--parse] [seedOrNetworkPassphrase]
{"seed":"SAYANWJGVJJU2EFM7VVW7LPUEDT64XCAFOLVNVXI2XK4T5MTL7GYKIII","address":"GCKMRRAOA5X6O5DHALNYE32KB2LNIEI5P2OXOSAZMDXIMBEQLCLMU3ZL"}

$ sebak key generate --format prettyjson [--parse] [seedOrNetworkPassphrase]
{
  "seed": "SAWPETN2L6YKGDRG5A5SVRNZ4KPBWFQ6LQP2OQS6OMCDGYUTROMCIBRZ",
  "address": "GDIWDGGNKUZEHWXH3DSCCED5TCD6E4HLDTDOJQDSVXVA6BCCWZNM5LJ4"
}

$ sebak key generate --format pretty [--parse] [seedOrNetworkPassphrase]
       Secret Seed: SB6BVDSRLGQ4AOB3ZTL2FSMGJJVYHQ6KEDLTWDFR25F5AOARNIISL6UL
    Public Address: GB5MBN4EGKO54A4GTHM5AL2ASSQRPNB3ESSHLXKPBVCPZN3AJL7QC4Z7

$ sebak key generate --format oneline [--parse] [seedOrNetworkPassphrase]
SAPGPM76SODAFPHJR2PQIAJTRWA5H7EOGRZAVYE32BDQ3AHYMETKM5FH GB2EJGMYJS3XBEWLVGPRJEDLBM75R35CMOVVRTMOGOQIDXKQTEBCOXPI

```

### CODE REVIEW

File: cmd/sebak/cmd/key/generate.go [hunk 1]
```diff
@@ -56,27 +107,22 @@ Network Passphrase: "{{ .networkPassphrase}}"{{ end }}
...
+	} else if fromSeed {
 		var kp keypair.KP
-		kp, err = keypair.Parse(flagInput)
-		if err != nil {
-			return
-		}
 
-		full = kp.(*keypair.Full)
+		if kp, err = keypair.Parse(seedOrNetworkPassphrase); err == nil {
+			full = kp.(*keypair.Full)
```
Comment (spikeekips) [inline cmd/sebak/cmd/key/generate.go]:
```
$ go run cmd/sebak/main.go  key generate --parse GC4K2MMPWBRAACDCBPTQPEV3CHIVZIM4K6RJZQFWXCNRQAD74S2T2CB3
panic: interface conversion: keypair.KP is *keypair.FromAddress, not *keypair.Full

goroutine 1 [running]:
boscoin.io/sebak/cmd/sebak/cmd/key.generateKP(0x7ffeefbff4de, 0x38, 0x7ffeefbff401, 0x38, 0x1, 0x7ffeefbff4de)
	/Users/spikeekips/workspace/blockchainos/sebak/src/boscoin.io/sebak/cmd/sebak/cmd/key/generate.go:121 +0x11c
boscoin.io/sebak/cmd/sebak/cmd/key.init.0.func1(0xc4200d3900, 0xc420160e00, 0x1, 0x2)
	/Users/spikeekips/workspace/blockchainos/sebak/src/boscoin.io/sebak/cmd/sebak/cmd/key/generate.go:67 +0x106
boscoin.io/sebak/vendor/github.com/spf13/cobra.(*Command).execute(0xc4200d3900, 0xc420160dc0, 0x2, 0x2, 0xc4200d3900, 0xc420160dc0)
	/Users/spikeekips/workspace/blockchainos/sebak/src/boscoin.io/sebak/vendor/github.com/spf13/cobra/command.go:766 +0x2c1
boscoin.io/sebak/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0x18768c0, 0x1876b20, 0xc420079f78, 0x100620c)
	/Users/spikeekips/workspace/blockchainos/sebak/src/boscoin.io/sebak/vendor/github.com/spf13/cobra/command.go:852 +0x30a
boscoin.io/sebak/vendor/github.com/spf13/cobra.(*Command).Execute(0x18768c0, 0x9, 0x0)
	/Users/spikeekips/workspace/blockchainos/sebak/src/boscoin.io/sebak/vendor/github.com/spf13/cobra/command.go:800 +0x2b
boscoin.io/sebak/cmd/sebak/cmd.Execute()
	/Users/spikeekips/workspace/blockchainos/sebak/src/boscoin.io/sebak/cmd/sebak/cmd/init.go:22 +0x2d
main.main()
	/Users/spikeekips/workspace/blockchainos/sebak/src/boscoin.io/sebak/cmd/sebak/main.go:8 +0x20
exit status 2
```
When `--parse` value is not secret seed...

File: cmd/sebak/common/encoder.go [hunk 2]
```diff
@@ -0,0 +1,31 @@
+package common
+
+import (
+	"encoding/json"
+	"gopkg.in/yaml.v2"
```
Comment (spikeekips) [inline cmd/sebak/common/encoder.go]: This needs to `dep ensure -add` :)

**Comment by Geod24** [comment]
Ping @zzim2x , there's 2 comments left from Minhyo

**Comment by Geod24** [comment]
Ping

**Comment by spikeekips** [comment]
@zzim2x Ping too!

**Comment by zzim2x** [comment]
already apply code review comments. @spikeekips

**Comment by outsideris** [comment]
ping @anarcher
